pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -2580,8 +2582,19 @@ private boolean isSystemTopic(String topic) {
         }
     }
 
+    public CompletableFuture<Void> deleteTopicPolicies(TopicName topicName) {
+        if (!pulsar().getConfig().isTopicLevelPoliciesEnabled()) {
+            return new CompletableFuture<>();
```

Comment:
Do you want to return a completed future?

Reply:
I'm not sure return completed future is the best way, so followed the origin code style, return completed fusture. I will modify it.

Reply:
Finally, I maintain the complete fusture way. I think use complete fusture is more suitable for this method. Cound you give some advice.

Reply:
@horizonzy I think you need a completed `CompletableFuture` right?

Reply:
Maybe the new `CompletableFuture` should be completed. Such as `            CompletableFuture.completedFuture(null);`

Reply:
right.

Reply:
sorry, I wasn't clear.
I was suggesting to return "CompletableFuture.completedFuture(null)"

Reply:
> sorry, I wasn't clear.
> I was suggesting to return "CompletableFuture.completedFuture(null)"

My fault, I misunderstand it. :)